datarepo-clienttests/src/main/java/common/utils/LogsUtils.java

```diff
@@ -18,11 +18,9 @@
   private static final Logger logger = LoggerFactory.getLogger(LogsUtils.class);
 
   // The log/metric timestamps are not exact and so trying to limit results to too small a window
-  // might be misleading
-  // This parameter sets the minimum time interval size. If the interval specified is less than
-  // this, then the code
-  // will expand the interval to the minimum size.
-  private static int minimumTimeRangeSizeInSeconds = 300; // = 5 minutes
+  // might be misleading This parameter sets the minimum time interval size. If the interval
```

Comment:
```suggestion
  // might be misleading. This parameter sets the minimum time interval size. If the interval
```

Reply:
I'll fix this in the follow-on PR. Thank you!